rca/utils/models.py

```diff
@@ -725,3 +725,21 @@ def clean(self):
             "Contact",
         )
     ]
+
+
+def get_listing_image(page):
+    """Global function to get a page listing image, should use the
+    listing_image if set, if not, check for a hero image
+
+    Args:
+        page (wagtail.code.Page): The page object
```

Comment:
Wagtail `Page` is unlikely to have a listing or hero image, maybe remove `(wagtail.code.Page)(wagtail.code.Page)`